rust/private/clippy.bzl

```diff
@@ -146,6 +160,10 @@ rust_clippy_aspect = aspect(
     fragments = ["cpp"],
     host_fragments = ["cpp"],
     attrs = {
+        "_capture_output": attr.label(
+            doc = "If true, write clippy output as a build output, rather than printing it, using the configured error_format",
```

Comment:
TBH I think the doc here is not needed. Will buildifier complain if we leave it out? If yes, or if you prefer to have a doc set, I'd go with "Value of the capture_clippy_output build setting".

Reply:
The linter complains. Changed to "Value of the capture_clippy_output build setting"